### DISCUSSION TITLE
Volume distribution for a backup configuration

### DISCUSSION THREAD

**Discussion by arthurzenika** [borgbase/vorta#806] [Ideas]
**Is your feature request related to a problem? Please describe.**
With a complex configuration for vorta/borg I have trouble finding what is using up the backup space, and what additional excluding patterns I could add. 

**Describe the solution you'd like**
Have a diskusage / baobab type visualisation of the data set that will backed up (using the exlusion patterns) so that a user can adjust those while configuring the profile. 

Tools for visual inspiration : https://wiki.gnome.org/action/show/Apps/DiskUsageAnalyzer https://windirstat.net/ https://en.wikipedia.org/wiki/Ncdu

**Describe alternatives you've considered**
Manage this sort of thing outside of vorta. But I don't know how. Or find a software that takes into account the .nobackup files or and exclude pattern.

**Comment by m3nu** [comment]
The backup will mostly mirror the actual files. So using one of the tools you linked to will solve part of the problem.

I realize that it doesn't account for having only selected backups or exclusions. In theory we could get the data from Borg's archive-list feature.

But then I'm not sure this is an urgent feature. If you like, it could still be added to the extract or diff dialogue. We'd just need to do add up all the files in subfolders.

**Comment by arthurzenika** [comment]
Thanks for taking the time to respond to this feature request.
